src/components/Overlays/Modal/Modal.tsx

```diff
@@ -108,7 +109,9 @@ export const Modal = forwardRef<HTMLDivElement, ModalProps>(({
           ref={ref}
           className={classNames(styles.wrapper, className)}
           {...restProps}
+          aria-describedby={undefined}
```

Comment:
Hey, thank you for your PR.

`aria-describedby` should be undefined without it. You also rewrite it for restProps in case someone wants to pass it, did you do it on purpose?